_pages/en_US/installing-boot9strap-(mset9-cli).txt

```diff
@@ -77,6 +77,8 @@ These instructions must be followed **EXACTLY**, so double-check EVERYTHING you
 1. Press (A) to launch System Settings
 1. Navigate to `Data Management` -> `Nintendo 3DS` -> `Extra Data` ([image](/images/screenshots/mset9/settings-extdata.png))
 1. **Do not press any buttons or touch the screen** - ensure that you see the Mii Maker icon
+    + If you have multiple pages of extra data, scroll to ensure Mii Maker is present, power off, then power on your console and resume from Section II Step 3, then stay on the first page for the next step
+    + If nothing loads and you see a spinning circle, you injected early. Insert the SD back into your PC, run the script again, choose your model/version and type 3 then press enter to Remove MSET9. Insert the SD back into your console, power it on and return to Section II Step 1
```

Comment:
This sequence may be better as additional nested bullet points for ease of readability

Reply:
So a bullet per line or is there a different formatting we're looking for?

+ If nothing loads and you see a spinning circle, you injected early.
+ Insert the SD back into your PC, run the script again, choose your model/version
+ type 3 then press enter to Remove MSET9. 
+ Insert the SD back into your console, power it on and return to Section II Step 1

Reply:
- If nothing loads and you see a spinning circle, you injected early:
    - Insert the SD card back into your PC
    - Rerun the script, selecting the same model/version that was used earlier
    - Type 3, then press Enter to remove MSET9
    - Reinsert the SD card into your console
    - Power on your console
    - Return to Section II Step 1
(Ideally there would be a way to condense this... my way of choice would be troubleshooting-style dropdowns but I think @lifehackerhansol and the translators would kill me if we implemented this)

Reply:
alternatively, given that we've now reworded things so that injecting early SHOULDN'T happen, we can leave this out and see how things go for a bit

Reply:
Do dropdowns even work nested this far? Like does Jekyll render that right?

If we have to do dropdowns, then whatever, as long as it doesn't crap the bed everywhere. But yeah, this one should be considered for the future seeing as we've done work into preventing that scenario.

Reply:
> alternatively, given that we've now reworded things so that injecting early SHOULDN'T happen, we can leave this out and see how things go for a bit

I've definitely seen less early injections after the rewording, checking the two channels, it looks like it is down to a few early injections a week instead of a few a day.

I would remove the early injection part at this point but still figure out some wording for the multiple page extra data thing.